_gsocproposals/2020/proposal_TMVAInference.md

```diff
@@ -0,0 +1,30 @@
+---
+title: TMVA Deep Learning Developments - Inference
+layout: gsoc_proposal
+project: TMVA
+year: 2020
+organization:
+  - CERN
+  - OProject
+---
+
+# Description
+
+Toolkit for Multivariate Analysis (TMVA) is a multi-purpose machine learning toolkit integrated into the ROOT scientific software framework, used in many particle physics data analysis and applications. We provide in TMVA the functionality for building deep neural networks including fully connected, convolutional and recurrent layers. 
+This project will focus on development of missing deep learning operations which will allow to build more complex networks within TMVA and with the aim to provide a fast inference engine. The operators we are planning to implement  will allow the merge of the output of different layers using concatenation or addition. 
+
+## Task ideas and expected results
+  * Support for concatenation and addition of layers
```

Comment:
Expand clarify the items and add clear deliverables

Reply:
@lmoneta